### PR TITLE
Add Roboflow to `Third-Party Tools` in AI & Vectors section

### PR DESCRIPTION
**Pull request by capjamesg**
## What kind of change does this PR introduce?

This PR adds a new guide that shows how to use Roboflow with Supabase in the `Third-Party Tools` section of the AI & Vectors section in the Supabase documentation/

## What is the current behavior?

N/A

## What is the new behavior?

A new documentation page appears with two tutorials:

1. How to save computer vision predictions in Supabase, and;
2. How to calculate CLIP vectors with Roboflow Inference and save results in Supabase.

Screenshot 2023-11-28 at 10 41 52

### CODE REVIEW

File: apps/docs/pages/guides/ai/integrations/roboflow.mdx [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+export const meta = {
+  id: 'ai-integration-roboflow',
+  title:
+    'Roboflow',
+  subtitle:
+    'Learn how to integrate Supabase with Roboflow, a tool for running fine-tuned and foundation vision models.',
+  breadcrumb: 'AI Integrations',
+}
+
+This guide will walk you through a basic example using [Roboflow Inference](https://inference.roboflow.com) to run inference on fine-tuned and foundation models. We will run inference and save predictions using an object detection model and [CLIP](https://github.com/openai/CLIP).
```
Comment (charislam) [inline apps/docs/pages/guides/ai/integrations/roboflow.mdx:12]:
```suggestion
This guide will walk you through a basic example using [Roboflow Inference](https://inference.roboflow.com) to run inference on fine-tuned and foundation models. You will run inference and save predictions using an object detection model and [CLIP](https://github.com/openai/CLIP).
```

Minor nit: in our [style guide](https://github.com/supabase/supabase/blob/master/apps/docs/CONTRIBUTING.md) we prefer `you` over `we`

File: apps/docs/pages/guides/ai/integrations/roboflow.mdx [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+
+For this tutorial, we have prepared two notebooks:
```
Comment (charislam) [inline apps/docs/pages/guides/ai/integrations/roboflow.mdx:14]:
```suggestion
This tutorial uses two Roboflow notebooks:
```

File: apps/docs/pages/guides/ai/integrations/roboflow.mdx [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+1. Save computer vision model predictions in Supabase
+2. Calculate and save CLIP embeddings in Supabase
+
+<DatabaseSetup />
+
+## Save Computer Vision Predictions
+
+Once you have a trained vision model, you need to create business logic for your application. In many cases, you may want to save inference results to a file.
```
Comment (charislam) [inline apps/docs/pages/guides/ai/integrations/roboflow.mdx:23]:
```suggestion
Once you have a trained vision model, you need to create business logic for your application. In many cases, you want to save inference results to a file. 
```

File: apps/docs/pages/guides/ai/integrations/roboflow.mdx [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+1. Save computer vision model predictions in Supabase
+2. Calculate and save CLIP embeddings in Supabase
+
+<DatabaseSetup />
+
+## Save Computer Vision Predictions
+
+Once you have a trained vision model, you need to create business logic for your application. In many cases, you may want to save inference results to a file. 
+
+Let's talk through how to run a vision model locally and save predictions to Supabase.
```
Comment (charislam) [inline apps/docs/pages/guides/ai/integrations/roboflow.mdx:25]:
```suggestion
The steps show you how to run a vision model locally and save predictions to Supabase.
```

File: apps/docs/pages/guides/ai/integrations/roboflow.mdx [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+
+### Preparation: Set up a Model
+
+Before we begin, you will need an object detection model trained on your data.
```
Comment (charislam) [inline apps/docs/pages/guides/ai/integrations/roboflow.mdx:29]:
```suggestion
Before you begin, you will need an object detection model trained on your data.
```

**Review by charislam — APPROVED** [review]
Thanks for the PR! Looks great.

Just a few minor nits/suggestions to bring this more in line with our style

**Comment by capjamesg** [comment]
@charislam I have incorporated all of your feedback into my latest commit.

**Comment by charislam** [comment]
looks great, thanks! can you run a quick `npm run format` so the merge checks will pass?

**Comment by capjamesg** [comment]
I have just run the command and committed my changes. Let me know if that works!

**Comment by charislam** [comment]
That's great, thank you!

**Pull request merged**